dwellingly/src/components/NavigationMenu/navigationMenu.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { 
+  faTicketAlt, 
+  faColumns, 
+  faPlusCircle, 
+  faUserCog, 
+  faBook, 
+  faUserAlt, 
+  faPhoneAlt, 
+  faCog 
+} from '@fortawesome/free-solid-svg-icons';
+import { useLocation, Link } from "react-router-dom";
+import PropTypes from 'prop-types';
+import "bulma/css/bulma.css";
+
+
+export class LogOutButton extends React.Component {
+  render() {
+    return (
+      <div className="is-logout-button">
+        <button className='button is-dark is-rounded is-small'>LOG OUT</button>
+      </div>
+    );
+  }
+}
+
+export const MenuLink = ({ icon, isBold, name, href }) => {
+
+  // look at the href prop and find the last, most specific piece of the path
+  // this piece can be used as the id for this MenuLink
+  let leafHrefString = href.split("/").splice(-1, 1)[0];
+
+  const loc = useLocation();
+  const locParts = loc.pathname.split("/");
+
+  // loop through the parts of the current location path (which are based on React Router in App)
+  // the current link is "active" if any part of the current path matches the href in our props
+  let isActiveLink = locParts.some(pathPart => pathPart === leafHrefString);
+  
+  const linkable = (leafHrefString === "add" || leafHrefString === "manage") ? "is-inactive-link" : "";
+  const classNameBase = "is-size-7 " + (isBold ? "has-text-weight-bold " : "") + linkable;
+  const linkDefaultColor = (leafHrefString === "reports") ? "has-text-grey" : "has-text-white";
+  const linkColor = (isActiveLink) ? "has-text-black" : linkDefaultColor;
+
+  return (
+    <li className="is-padding">
+      <Link to={href} className={`${classNameBase} ${linkColor}`}>
+        {icon && 
+          <span className="icon is-small">
+            <FontAwesomeIcon icon={icon} />
+          </span>
+        }
+        <span className="is-menu-link">
+          {name}
+        </span>
+      </Link>
+    </li>
+  );
+};
+
+MenuLink.propTypes = {
+  icon: PropTypes.object,
+  isBold: PropTypes.bool,
+  name: PropTypes.string.isRequired,
+  href: PropTypes.string.isRequired,
+}
+
+export const NavMenu = () => {
+  const loc = useLocation();
+  if (loc.pathname === "/login" || loc.pathname === "/signup" || loc.pathname === "/terms"){
+    return null;
+  }
+
+  return (
+    <div className="is-hidden-mobile is-sidebar-menu bg-blue">
+      <div className="menu">
+        <ul className="menu-list">
+          <div className="is-username-link">
```

Comment:
I think we could make this div a bit bigger! It seems small given the space it's in, compared to the mockup 
![image](https://user-images.githubusercontent.com/8600792/77833214-dff81800-70f8-11ea-8d6d-45e4197037b6.png)